easybuild/easyconfigs/m/minimap2/minimap2-2.20-GCCcore-10.3.0.eb

```
@@ -37,17 +37,18 @@ files_to_copy = [
     (['lib%(name)s.a'], 'lib'),
     (['*.h'], 'include'),
     'LICENSE.txt', 'NEWS.md', 'README.md',
-    (['%(name)s.1'], 'share/man/man1')
+    (['%(name)s.1'], 'share/man/man1'),
+    (['test/MT-human.fa', 'test/MT-orang.fa'], 'test'),
 ]
 
 sanity_check_paths = {
     'files': ['bin/%(name)s', 'lib/lib%(name)s.a'],
-    'dirs': ['include']
+    'dirs': ['include', 'test']
 }
 
 sanity_check_commands = [
     "minimap2 --help",
-    "cd %(builddir)s/minimap2-%(version)s && minimap2 -a test/MT-human.fa test/MT-orang.fa > test.sam",
+    "cd %(installddir)s && %(name)s -a test/MT-human.fa test/MT-orang.fa > test.sam",
```

```suggestion
    "cd %(installdir)s && minimap2 -a test/MT-human.fa test/MT-orang.fa > test.sam",
```
Fix typo and see if that was also causing the CI to complain about using `minimap` instead of `%(name)s`